docs/eventing/sugar/README.md

```diff
@@ -0,0 +1,92 @@
+Knative Eventing Sugar Controller will react to special labels and annotations
+to produce or control eventing resources in a cluster or namespace.
+
+## Installing
+
+The following command installs the Eventing Sugar Controller:
+
+```bash
+kubectl apply --filename {{< artifact repo="eventing" file="eventing-sugar-controller.yaml" >}}
+```
+
+## Automatic Broker Creation
+
+One way to create a Broker is to manually apply a resource to a cluster using
+the default settings:
+
+```shell
+kubectl create -f - <<EOF
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: default
+  namespace: default
+EOF
+```
+
+There might be cases where automated Broker creation is desirable, such as on
+namespace creation, or on Trigger creation. The Sugar controller enables those
+use-cases:
```

Comment:
I thought there was a way to have it autogen a Broker for any new namespace without adding an annotation? opt-out?

Reply:
I was thinking that the namespace version of sugaring seemed more useful as an opt-out, since you'd need edit on namespaces to be able to apply the label.

Reply:
I think you are thinking of the SinkBinding @vaikas.

Reply:
> I was thinking that the namespace version of sugaring seemed more useful as an opt-out, since you'd need edit on namespaces to be able to apply the label.

We can think about that as a followup feature, but the thinking if if you are provided a namespace, and have no control over the labels, it is an option when the namespace is created provided by the admin of that cluster.